Code/GraphMol/MolDraw2D/MolDraw2D.h

```diff
@@ -306,6 +306,9 @@ struct RDKIT_MOLDRAW2D_EXPORT MolDrawOptions {
       false;  // if all specified stereocenters are in a single StereoGroup,
               // show a molecule-level annotation instead of the individual
               // labels
+  bool blackWedgeBonds = false; // if true wedged and dashed bonds are drawn
```

Comment:
Maybe call this `singleColouWedgeBonds`?

Reply:
This is now done

Reply:
> This is now done

Thanks for fixing the typo in my proposed name. :-)

Reply:
I did actually copy/paste from here, and only caught it when staring at the diff before committing.